activities/codebase-auditing/open-audits.md

```diff
@@ -0,0 +1,22 @@
+---
+type: Guide
+explains: how we do audits in the open
+redirect_from:assessments
+    - activities/codebase-stewardship/open-assessment
+---
+
+# How we do audits in the open
+
+This guide is intended to help codebase stewards start audits on a codebase.
+
+## Principles
+
+As much as possible, audit should take place in the open and be done together with the community of the codebase.
+
+## Steps
+
+1. Get explicit approval to start an open audit by asking the maintainer. Consider communicating that audit is starting, preferrably by encouraging the maintainer to do it, and possibly on [the blog](https://blog.publiccode.net) as well.
+2. Create an issue in the repository for the codebase using the [review template](review-template.md).
+3. Start auditing.
```

Comment:
This text does not make clear that the audits are done by a collaboration of one or more existing key contributors and one or more codebase stewards.

People new or external to a codebase (even experienced codebase stewards) would have a very hard time evaluating contextually sensitive questions on their own.